### ISSUE TITLE
x86 windows crash

### ISSUE THREAD

**Issue by ggpabuk** [TolikPylypchuk/SharpHook#10]
https://www.youtube.com/watch?v=eM8xPIZ57tA

When compiling under x86, the cursor starts to move jerkily, then the program crashes

**Comment by TolikPylypchuk** [comment]
Hi! I've tried running the x86 version of the sample app on Windows and it does fail with an `AccessViolationException`:

```
---------- SharpHook Sample ----------

System info:
DEBUG: hook_get_auto_repeat_rate [129]: SPI_GETKEYBOARDSPEED: 31.
Fatal error. 0xC0000005
   at SharpHook.Native.UioHook.GetAutoRepeatRate()
   at SharpHook.Sample.Program.PrintSystemInfo()
   at SharpHook.Sample.Program.Main()
```

Unfortunately, I don't know why it happens and how to fix it as I'm not knowledgeable enough on the internals of P/Invoke :(

Maybe, I'll return to this issue in the future, but for now some help will be appreciated.

**Comment by TolikPylypchuk** [comment]
Ok, turns out that it was because of an incorrect calling convention for the callback delegates. This issue will be fixed in version 3.0.2 which I'll release later today.

**Comment by TolikPylypchuk** [comment]
Version 3.0.2 is released and this issue is fixed there.
